cmd/argo/commands/clustertemplate/create.go

```diff
@@ -25,6 +25,15 @@ func NewCreateCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "create FILE1 FILE2...",
 		Short: "create a cluster workflow template",
+                Example: `# Create a Cluster Workflow Template:
+argo cluster-template create FILE1
+	
+# Create a Cluster Workflow Template with relaxed validation:
+argo cluster-template create FILE1 --strict=false
+	  
+# Create a Cluster Workflow Template and print it as JSON:
+argo cluster-template create FILE1 --output json
```

Comment:
```suggestion
# Create a Cluster Workflow Template and print it as YAML:
  argo cluster-template create FILE1 --output yaml
```

- per my [previous comment](https://github.com/argoproj/argo-workflows/pull/12101#discussion_r1375493101), let's use a more frequently used k8s format like YAML
- 2 space indent for the command, same as above

Reply:
also I mentioned [before](https://github.com/argoproj/argo-workflows/pull/12101#discussion_r1375492256), can you re-order this to be above `--strict`? `--output` is first alphabetically and I imagine may be more frequently used as well of the two flags